data/meson.build

```
@@ -0,0 +1,47 @@
+subdir('ui')
+
+desktop_file = i18n.merge_file(
+    input: 'pcmanfm.desktop.in',
+    output: 'pcmanfm.desktop',
+    type: 'desktop',
+    po_dir: '../po',
+    install: true,
+    install_dir: get_option('prefix') / get_option('datadir') / 'applications'
+)
+
+desktop_file_pref = i18n.merge_file(
+    input: 'pcmanfm-desktop-pref.desktop.in',
+    output: 'pcmanfm-desktop-pref.desktop',
+    type: 'desktop',
+    po_dir: '../po',
+    install: true,
+    install_dir: get_option('prefix') / get_option('datadir') / 'applications'
+)
+
+desktop_utils = find_program('desktop-file-validate', required: false)
+if desktop_utils.found()
+    test('Validate desktop file', desktop_utils,
+        args: [desktop_file]
+    )
+endif
+
+if desktop_utils.found()
+    test('Validate desktop file', desktop_utils,
+       args: [desktop_file_pref]
+    )
+endif
```

The following should also work:

```suggestion
if desktop_utils.found()
    test('Validate desktop file', desktop_utils,
        args: [desktop_file, desktop_file_pref]
    )
endif
```